pydantic_ai_slim/pydantic_ai/ext/aci_tools.py

```diff
@@ -0,0 +1,82 @@
+from aci import ACI
+
+def clean_schema(schema):
+    if isinstance(schema, dict):
+        # Remove non-standard keys (e.g., 'visible')
+        return {k: clean_schema(v) for k, v in schema.items() if k not in {'visible'}}
+    elif isinstance(schema, list):
+        return [clean_schema(item) for item in schema]
+    else:
+        return schema
+
+def tool_from_aci(aci_function: str,
+                  linked_account_owner_id: str
+                  ) -> Tool:
+    """Creates a Pydantic tool proxy from a ACI tool.
+
+    Args:
+        aci_function: The ACI function to use.
+        linked_account_owner_id: The ACI user ID to execute the function on behalf of.
+
+    Returns:
+        A Pydantic tool that corresponds to the LangChain tool.
+    """
+    aci = ACI()
+    # print(aci_function)
+    function_definition = aci.functions.get_definition(aci_function)
+    # print(function_definition)
+
+    function_name = function_definition["function"]['name']
+    function_description = function_definition["function"]['description']
+    inputs = function_definition["function"]['parameters']
+
+    json_schema = {
+    'additionalProperties': inputs.get('additionalProperties', False),
+    'properties': inputs.get('properties', {}),
+    'required': inputs.get('required', []),
+    'type': inputs.get('type', 'object'), # Default to 'object' if not specified
+    }
+
+    # Clean the schema
+    json_schema = clean_schema(json_schema)
+
+    def implementation(**kwargs) -> str:
```

Comment:
Technically this can take args as well, so like in `langchain.py` let's accept them but raise an error if they're provided.